src/og/Globe.js

```diff
@@ -178,7 +186,8 @@ class Globe {
                 maxEqualZoomAltitude: options.maxEqualZoomAltitude,
                 minEqualZoomAltitude: options.minEqualZoomAltitude,
                 minEqualZoomCameraSlope: options.minEqualZoomCameraSlope,
-                loadingBatchSize: options.loadingBatchSize
+                loadingBatchSize: options.loadingBatchSize,
+                quadTreeStrategyPrototype: options.quadTreeStrategyPrototype
```

Comment:
also, what if this is not provided?

Reply:
I've changed the name to maxLoadingRequests, by default it is equal 12 it is described in jsdoc head in Globe module `@param {Number} [options.maxLoadingRequests=12] `